src/stream/src/executor/receiver.rs

```diff
@@ -81,9 +87,18 @@ impl Executor for ReceiverExecutor {
         let mut status = self.status;
         let metrics = self.metrics.clone();
         let actor_id_str = self.actor_id.to_string();
+        let upstream_fragment_id_str = self.upstream_fragment_id.to_string();
+        let mut start_time = minstant::Instant::now();
```

Comment:
The start time for the first message seems incorrect here: it'll record before we start to poll the stream of this executor. Maybe we need to rewrite it with `futures_async_stream`.